canonical-web-and-design/snapcraft.io
# Convert first pages to typescript

## Done
Added tests and converted `NoStores` and `NotAuthorized` to TypeScript

## QA
- Check that the code builds
- Check that the tests pass, e.g. `dotrun exec yarn test-js static/js/brandstore`

## Issues
Fixes #3771
Fixes #3768
Fixes #3769
Fixes #3770
Fixes #3729
